src/lightning/fabric/fabric.py

```diff
@@ -746,6 +746,22 @@ def load(
                 state[k] = unwrapped_state[k]
         return remainder
 
+    def load_raw(self, path: Union[str, Path], obj: Union[nn.Module, Optimizer], strict: bool = True) -> None:
```

Comment:
Point of discussion:

1. Should this be a separate method or the functionality combined in Fabric.load?
2. Alternative names considered: load_raw, load_object, load_torch, load_from_torch

Reply:
(1) I like the current approach (i'm biased though)
(2) I prefer `_torch` a little bit - raw could be interpreted as bytes or something similar.

Reply:
I also like the current one.